js/settings.js

```diff
@@ -1,32 +1,29 @@
 /* global Mail */
-$(document).ready(function(){
+$(document).ready(function() {
 
-    // delete account in settings
-	$(document).on('click', '.mailaccount-list .action.delete', function () {
+	// delete account in settings
+	$(document).on('click', '.mailaccount-list .action.delete', function() {
 
-        $(this).removeClass('icon-delete').addClass('icon-loading-small');
+		$(this).removeClass('icon-delete').addClass('icon-loading-small');
 
 		var accountId = $(this).parent().parent().data('account-id');
 
 		$.ajax(OC.generateUrl('/apps/mail/accounts/{accountId}'), {
 			data: {accountId:accountId},
 			type: 'DELETE',
-			success: function () {
-                // reload the complete page
-                // TODO should only reload the app nav/content
-                window.location.reload();
+			success: function() {
+				// reload the complete page
+				// TODO should only reload the app nav/content
+				// window.location.reload();
```

Comment:
This is the only actual code change - commenting out the line. Was this really intended?